fedramp-integrated-inventory-workbook/.gitignore

```diff
@@ -0,0 +1,116 @@
+# Ignore MacOS folder
```

Comment:
this gitignore should probably just be merged into the one at the top of the repository unless there's a good reason to maintain a separate one?

Reply:
Combined the two .gitignores, only took the required statements from the fedramp's .gitignore.